client/state/sites/actions.js

```diff
@@ -27,6 +27,7 @@ import {
 	SITE_PLUGIN_UPDATED,
 } from 'state/action-types';
 import { SITE_REQUEST_FIELDS, SITE_REQUEST_OPTIONS } from 'state/sites/constants';
+import 'state/data-layer/wpcom/sites/gutenberg';
```

Comment:
@dmsnell I'm confused as to why this is needed here; moving it to `state/actions/editor/actions.js` is ineffective (the API call will no longer fire). What am I missing here?..

Reply:
maybe we should talk about this in a screenshare. the import statement should belong in the file which can dispatch `EDITOR_TYPE_SET` actions, not here. that is, the linkage is between things that can dispatch actions of a given type and the handlers that listen to that type.

something sounds fishy here.

Reply:
Did you two happen to sort this out in one of your screenshares? 😄

Reply:
this import will now go into the new `client/state/selected-editor/actions.js` file 👍